src/components/settings/settings.component.tsx

```diff
@@ -4,12 +4,14 @@ import { useTheme } from '../../providers';
 import { HotkeysSettings } from './hotkeys-settings.component';
 import { GeneralSettings } from './general-settings.component';
 import { ThemesSettings } from './themes-settings.component';
+import {SpotifySettings} from "./spotify-settings.component";
```

Comment:
Please add spaces and use `'` queries 

`import { SpotifySettings } from './spotify-settings.component';`